src/mmpars.c

```diff
@@ -5645,272 +5649,271 @@ vstring readInclude(const char *fileBuf, long fileBufOffset,
               cmdPos1 - 1 - startOffset);
           g_IncludeCall[saveInclCalls - 1].current_line = 0;
           aftInclLineNum = befInclLineNum + countLines(newFileBuf
-              + cmdPos1/*start at (cmdPos1+1)th character*/,
+              + cmdPos1, // start at (cmdPos1+1)th character
               cmdPos2 - cmdPos1 - 1);
           g_IncludeCall[saveInclCalls].current_line = aftInclLineNum;
           parentLineNum = aftInclLineNum;
 
-          /* Call recursively to expand includes in the included source */
-          /* Start at line 1 since source is in external file */
+          // Call recursively to expand includes in the included source.
+          // Start at line 1 since source is in external file.
           free_vstring(inclSource);
           inclSource = readInclude(tmpSource,
-              fileBufOffset + cmdPos1 - 1 + (long)strlen(inclPrefix), /*new offset*/
-              /*includeFn,*/ includeFn,
-              &inclSize /*input/output*/, 1/*parentLineNum*/, &(*errorFlag));
-
-          oldInclSize = cmdPos2 - cmdPos1; /* Includes old prefix and suffix */
-          /* "$( Begin $[...$] $)" must have a whitespace
-             after it; we use a newline.  readFileToString will ensure a
-             newline at its end.  We don't add whitespace after
-             "$( End $[...$] $)" but reuse the whitespace after the original
-             "$[...$]". */
+              fileBufOffset + cmdPos1 - 1 + (long)strlen(inclPrefix), // new offset
+              /* includeFn, */ includeFn,
+              &inclSize /* input/output */, 1 /* parentLineNum */, &(*errorFlag));
+
+          oldInclSize = cmdPos2 - cmdPos1; // Includes old prefix and suffix
+          // "$( Begin $[...$] $)" must have a whitespace
+          // after it; we use a newline.  readFileToString will ensure a
+          // newline at its end.  We don't add whitespace after
+          // "$( End $[...$] $)" but reuse the whitespace after the original
+          // "$[...$]".
           let(&newFileBuf, cat(left(newFileBuf, cmdPos1 - 1),
               inclPrefix, inclSource, inclSuffix,
               right(newFileBuf, cmdPos2), NULL));
           *size = *size - (cmdPos2 - cmdPos1) + (long)strlen(inclPrefix)
               + inclSize + (long)strlen(inclSuffix);
           newInclSize = (long)strlen(inclPrefix) + inclSize +
-                (long)strlen(inclSuffix); /* Includes new prefix and suffix */
-          /* Adjust starting position for next inclusion search (which will
-             be at the start of the included file continuing into the remaining
-             parent file) */
-          startOffset = cmdPos1 + newInclSize - 1; /* -1 since startOffset is
-              0-based but cmdPos2 is 1-based */
-              /*
-              + inclSize  /@ Use instead of strlen for speed @/
-              + (long)strlen(inclSuffix) */
-              ;  /* -1 since startOffset is 0-based but cmdPos1 is 1-based */
```

Comment:
Note: it seems there was a misplaced semicolon here, so I moved it in the comment.

Reply:
I think the comment is meant to say:
```c
          startOffset = cmdPos1 + newInclSize - 1 // -1 since startOffset is 0-based but cmdPos2 is 1-based
              // + inclSize  // Use instead of strlen for speed
              // + (long)strlen(inclSuffix) // -1 since startOffset is 0-based but cmdPos1 is 1-based
              ;
```
which is to say, the two commented out lines are additional addends on the `startOffset =` line and the semicolon on that line is the misplaced one. (I am not proposing this precise formatting, but something that gets the same idea across is fine. Probably the easiest thing to format is separate `startOffset += inclSize;` statements which are commented out.)

Reply:
I committed the change, is this formatting ok? https://github.com/metamath/metamath-exe/pull/152/commits/c5262300bb0103eb77995d154f2849c09597483d